modes/config-mode/src/ecgMode.js

```diff
@@ -0,0 +1,201 @@
+import { hotkeys } from '@ohif/core';
```

Comment:
The ecg mode is going to be the example for how to load extensions dynamically, so that the only thing in the code mode setup is extensions required and a flag as to applicability.  Everything else comes from the extension object itself.